lib/tasks/whitehall_importer.rb

```diff
@@ -120,37 +127,93 @@ def create_edition(document, translation, whitehall_edition, edition_number)
         created_at: whitehall_edition["created_at"],
       )
 
-      Edition.create!(
+      edition = Edition.create!(
         document: document,
         number: edition_number,
         revision_synced: true,
         revision: revision,
-        status: Status.new(
-          state: state(whitehall_edition),
-          revision_at_creation: revision,
-        ),
+        status: initial_status(whitehall_edition, revision),
         current: whitehall_edition["id"] == most_recent_edition["id"],
         live: live?(whitehall_edition),
         created_at: whitehall_edition["created_at"],
         updated_at: whitehall_edition["updated_at"],
         created_by_id: user_ids[first_author["whodunnit"]],
         last_edited_by_id: user_ids[last_author["whodunnit"]],
       )
+
+      set_current_status(whitehall_edition, edition) if whitehall_edition["state"] == "withdrawn"
```

Comment:
I wonder if we can give this a clearer name than current_status? maybe we should go with set_withdrawn_status to make it more explicit. Without changes it's a bit confusing why a withdrawn edition gets a current_status but nothing else does.